upgrade-notes/backend_20241205_151830.md

```diff
@@ -0,0 +1,7 @@
+#### Admin Grid performance improvements ([#2460](https://github.com/shopsys/shopsys/pull/{pullRequestId}))
+
+- Modified grid so that adding NULL FIRST / NULLS LAST is optional now, because it is much more time-consuming.
+- QueryBuilderWithRowManipulatorDataSource called callback for each item separately without the possibility of working with all items. Modified callback so that instead of calling ID = X for each item, ID IN (X, X, X) could be called only once. See example implementation in ProductControllet. In Callback function it is possible to request the entire result from query as second parameter.
+- Grid pulls out entire product (entity): Query is modified to return only the data that will be displayed later. Instead of entity, associated field is returned.
+- Search using LIKE cannot use B-tree index over columns with NORMALIZED function and LIKE condition %text%. Instead of standard B-tree index, new Trigram indexes were added on columns using LIKE conditions in Product, Order and Customer grids.
```

Comment:
I would rather see this as a description of the pull request on GitHub but I believe I can live with them here as well. Let's talk about it, please